### PR TITLE
Enable the organization shared user op event listener

### PR DESCRIPTION
**Pull request by ShanChathusanda93**
## Purpose
* $subject

### CODE REVIEW

File: components/org.wso2.carbon.identity.organization.user.invitation.management/src/main/java/org/wso2/carbon/identity/organization/user/invitation/management/InvitationCoreServiceImpl.java [hunk 1]
```diff
@@ -353,6 +354,13 @@ public boolean deleteInvitedUserAssociation(String userId, UserStoreManager user
         try {
             authenticatedOrganizationId = organizationManager.resolveOrganizationId(authenticatedTenant);
         } catch (OrganizationManagementException e) {
+            // This is to handle the scenario where the tenant is not an organization.
```
Comment (AnuradhaSK) [inline components/org.wso2.carbon.identity.organization.user.invitation.management/src/main/java/org/wso2/carbon/identity/organization/user/invitation/management/InvitationCoreServiceImpl.java:357]:
```suggestion
            // This is to handle the scenario where the tenant is not modeled as an organization.
```

File: components/org.wso2.carbon.identity.organization.user.invitation.management/src/main/java/org/wso2/carbon/identity/organization/user/invitation/management/InvitationCoreServiceImpl.java [hunk 2]
```diff
@@ -353,6 +354,13 @@ public boolean deleteInvitedUserAssociation(String userId, UserStoreManager user
...
+            if (e.getErrorCode().equals(ERROR_CODE_ORGANIZATION_NOT_FOUND_FOR_TENANT.getCode())) {
```
Comment (AnuradhaSK) [inline components/org.wso2.carbon.identity.organization.user.invitation.management/src/main/java/org/wso2/carbon/identity/organization/user/invitation/management/InvitationCoreServiceImpl.java:358]:
```suggestion
            if (ERROR_CODE_ORGANIZATION_NOT_FOUND_FOR_TENANT.getCode().equals(e.getErrorCode())) {
```

File: components/org.wso2.carbon.identity.organization.user.invitation.management/src/main/java/org/wso2/carbon/identity/organization/user/invitation/management/InvitationCoreServiceImpl.java (hunk 1 above)
Comment (ShanChathusanda93) [reply components/org.wso2.carbon.identity.organization.user.invitation.management/src/main/java/org/wso2/carbon/identity/organization/user/invitation/management/InvitationCoreServiceImpl.java:357]: Changed

File: components/org.wso2.carbon.identity.organization.user.invitation.management/src/main/java/org/wso2/carbon/identity/organization/user/invitation/management/InvitationCoreServiceImpl.java (hunk 2 above)
Comment (ShanChathusanda93) [reply components/org.wso2.carbon.identity.organization.user.invitation.management/src/main/java/org/wso2/carbon/identity/organization/user/invitation/management/InvitationCoreServiceImpl.java:358]: Flipped

**Review by AnuradhaSK — APPROVED** [review]

**Pull request merged**